src/probe/probe.c

```diff
@@ -759,7 +759,6 @@ static uint32_t probe_gen_format(uint32_t frame_fmt, uint32_t rate,
 	default:
 		tr_err(&pr_tr, "probe_gen_format(): Invalid frame format specified = 0x%08x",
 		       frame_fmt);
-		assert(false);
```

Comment:
I would also say this is not a FW fatal condition so should be removed.

Reply:
The reason behind dropping this assert() is the way its implemented now. It would require some problematic symbols to be exported from Zephyr core. So, since this is not really a show stopper, its easier to just driop this assert().